package-lock.json

```diff
@@ -1794,7 +1794,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I don't see a corresponding change in `package.json`; were these changes intentional, or an accident of clashing npm versions?